theseus/geometry/se3.py

```diff
@@ -123,7 +122,7 @@ def update_from_x_y_z_quaternion(self, x_y_z_quaternion: torch.Tensor):
 
     def update_from_rot_and_trans(self, rotation: SO3, translation: Point3):
         if rotation.shape[0] != translation.shape[0]:
-            raise ValueError("rotation and translation must have the same size.")
+            raise ValueError("rotation and translation must have the same batch size.")
```

Comment:
Flagging this for the scenario when a user might pass rotation and translation without a batch dim.

Reply:
Since inputs are `SO3` and `Point3`, they would already have a batch dim here, but we should add some type checks.

Reply:
@fantaosha @luisenp was this addressed before merging? If not, maybe add to an active PR.

Reply:
Yes, I meant a type check. But I don't think it's super important to have this, it'd be mostly for the user to get a less confusing error message, but I don't expect this function to be used a lot directly.